File: Assignments/Assignment 35/C Codes/program35_4.c
// Write a program which accepts a string from user and check it contains vowel or not.

#include<stdio.h>
#include<stdbool.h>

bool Difference(char str[])
{
    bool bAns = false;

    while(*str != '\0')
    {
        if(*str == 'A' ||*str == 'E' ||*str == 'I' ||*str == 'O' ||*str == 'U' ||
           *str == 'a' ||*str == 'e' ||*str == 'i' ||*str == 'o' ||*str == 'u')
        {
            bAns = true;
            break;
        }
        else
        {
            bAns = false;
        }
        str++;
    }

    return bAns;
}

int main()
{
    char Arr[50] = {'\0'};
    bool bRet = 0;

    printf("Enter String : \n");
    scanf("%[^'\n']s", Arr);

    bRet = Difference(Arr);

    if(bRet == true)
    {
        printf("It has Voewl\n");
    }
    else
    {
        printf("It has no Voewl\n");
    }

    return 0;
}

File: Assignments/Assignment 35/Java Codes/program35_4.java
// Write a program which accepts a string from user and check it contains vowel or not.

import java.util.*;

class Test 
{
    boolean Difference(String str)
    {
        char Arr[] = str.toCharArray(); 
        int iCnt  = 0; 
        boolean bAns =  false;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)       
        {
            if(Arr[iCnt] == 'A' ||Arr[iCnt] == 'E' ||Arr[iCnt] == 'I' ||Arr[iCnt] == 'O' ||Arr[iCnt] == 'U' ||
           Arr[iCnt] == 'a' ||Arr[iCnt] == 'e' ||Arr[iCnt] == 'i' ||Arr[iCnt] == 'o' ||Arr[iCnt] == 'u')
            {
                bAns = true;
                break;
            }
        }

        return bAns;
    }
}

class program35_4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        boolean bRet = false;

        System.out.println("Enter String : ");
        String Arr = sobj.nextLine();

        Test tobj = new Test();

        bRet = tobj.Difference(Arr);

        if(bRet == true)
        {
            System.out.println("It has Vowels");
        }
        else
        {
            System.out.println("It has no Vowels");
        }
    }
}